src/Core/Framework/Increment/IncrementGatewayRegistry.php

```diff
@@ -6,7 +6,7 @@
 use Shopware\Core\Framework\Log\Package;
 
 /**
- * @internal - Used internally for Increment pattern
```

Comment:
should stay internal, nothing changed here, there are other extension patterns to hook into this

Reply:
but you are not able to get access to a configured pool when you do not use this registry. Why having a registry when saying "use the direct implementation"